k2/csrc/ragged_utils_inl.h

```diff
@@ -0,0 +1,34 @@
+/**
+ * @brief
+ * ragged_utils_inl
+ *
+ * @note
+ * This is to be included only from ragged_utils.h.
+ *
+ * @copyright
+ * Copyright (c)  2020  Xiaomi Corporation (authors: Daniel Povey
+ *                                                   Haowen Qiu)
+ *                      Mobvoi Inc.        (authors: Fangjun Kuang)
```

Comment:
Ah, I think we can add Fangjun's copyright under `Xiaomi Corporation` now.

Reply:
mm ok, lets have him do that when he modifies files, I am trying to keep these to just the list of those who contributed to that file as a matter of general principle.
if it turns out we don't need this file I will delete it.